src/main/java/ua/artcode/udiary/dao/RecordDaoJsonImpl.java

```diff
@@ -11,9 +12,13 @@
 
 public class RecordDaoJsonImpl implements RecordDao {
 
-    private Gson gson = new GsonBuilder().setPrettyPrinting().create();
+    private Gson gson;
     private static final String RECORDS_PATH = "./src/ua/artcode/udiary/resources/record/records.txt";
```

Comment:
get from the properties

Reply:
RecordDao json will be changed. I will fix it after that.